solutions/src/main/java/ru/sberbank/school/task02/ExtendedFxConversionServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+/**
+ * Created by Mart
+ * 10.05.2019
+ **/
+public class ExtendedFxConversionServiceImpl implements ExtendedFxConversionService {
+    @Override
+    public Optional<BigDecimal> convertReversed(ClientOperation operation, Symbol symbol, BigDecimal amount, Beneficiary beneficiary) {
+        return Optional.empty();
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        return null;
+    }
+
+    @Override
+    public Optional<BigDecimal> convertReversed(ClientOperation operation, Symbol symbol, BigDecimal amount, double delta, Beneficiary beneficiary) {
```

Comment:
Line is longer than 120 characters (found 149).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-852104015>*